permissions/plugin-cloudify.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: "cloudify"
+github: "jenkinsci/cloudify-plugin"
+paths:
+- "io/jenkins/plugins/cloudify-plugin"
+developers:
+- "isaac_s"
+- "cosmo_admin"
+- "earthmant"
+- "alexmolev"
+security:
+  contacts:
+    jira: cosmo_admin
+    email: devops@cloudify.co
```

Comment:
```suggestion
```

This shouldn't be needed since cosmo_admin is already in the developers list.

Reply:
OK. Removed.